server/src/test/java/org/elasticsearch/index/mapper/DateFieldTypeTests.java

```diff
@@ -167,20 +170,19 @@ public void testTermQuery() {
                 new IndexSettings(IndexMetaData.builder("foo").settings(indexSettings).build(),
                         indexSettings),
                 null, null, null, null, null, xContentRegistry(), writableRegistry(), null, null, () -> nowInMillis, null);
-        MappedFieldType ft = createDefaultFieldType();
-        ft.setName("field");
+        MappedFieldType ft = createNamedDefaultFieldType();
         String date = "2015-10-12T14:10:55";
         long instant = DateFieldMapper.DEFAULT_DATE_TIME_FORMATTER.parser().parseDateTime(date).getMillis();
         ft.setIndexOptions(IndexOptions.DOCS);
         Query expected = new IndexOrDocValuesQuery(
-                LongPoint.newRangeQuery("field", instant, instant + 999),
```

Comment:
this is a horrible name, it doesnt mean anything especially when our tests get a bit more complicated than just a mapping with one field, eg like my new alias sub classes of this test.